### PR TITLE
Clarify auth tag lens

### PR DESCRIPTION
**Pull request by murillo128**

### CODE REVIEW

**Comment by murillo128** [comment]
@bifurcation @youennf should we use the 64 bits auth length for video or a 80 bits length one for AES_CM_128_HMAC_SHA256?

File: draft-omara-sframe.md [hunk 1]
```diff
@@ -458,23 +458,19 @@ o [Optional] A signature algorithm
 This document defines the following ciphersuites:
 
 
-| Value  | Name                           | Nh | Nk | Nn | Reference |
-|:-------|:-------------------------------|:---|:---|:---|:----------|
-| 0x0001 | AES\_CM\_128\_HMAC\_SHA256\_8  | 32 | 16 | 12 | RFC XXXX  |
-| 0x0002 | AES\_CM\_128\_HMAC\_SHA256\_4  | 32 | 16 | 12 | RFC XXXX  |
-| 0x0003 | AES\_GCM\_128\_SHA256          | 32 | 16 | 12 | RFC XXXX  |
-| 0x0004 | AES\_GCM\_256\_SHA512          | 64 | 32 | 12 | RFC XXXX  |
+| Value  | Name                           | Nh | Nk | Nn |Auth Tag Len | Reference |
```
Comment (bifurcation) [inline draft-omara-sframe.md:461]: Should probably clarify that this is in bits.

File: draft-omara-sframe.md [hunk 2]
```diff
@@ -458,23 +458,19 @@ o [Optional] A signature algorithm
...
+|:-------|:-------------------------------|:---|:---|:---|:------------|:----------|
+| 0x0001 | AES\_CM\_128\_HMAC\_SHA256\_64 | 32 | 16 | 12 |          64 | RFC XXXX  |
+| 0x0002 | AES\_CM\_128\_HMAC\_SHA256\_32 | 32 | 16 | 12 |          32 | RFC XXXX  |
+| 0x0003 | AES\_GCM\_128\_SHA256          | 32 | 16 | 12 |         128 | RFC XXXX  |
+| 0x0004 | AES\_GCM\_256\_SHA512          | 64 | 32 | 12 |         256 | RFC XXXX  |
```
Comment (bifurcation) [inline draft-omara-sframe.md:466]: The auth tag here is still 128 bits.

Comment (murillo128) [reply draft-omara-sframe.md:466]: The rest of values (Nh/NK/Nn) are in bytes, right?

Comment (bifurcation) [reply draft-omara-sframe.md:466]: Correct.  So either the auth tag length should be in bytes, or we should clarify that it's in bits.  (We could also assign it a cute name like the other columns, say `Nt`.)  In any case, AES-128-GCM and AES-256-GCM both have a 16-byte / 128-bit tag.

**Comment by youennf** [comment]
This seems partially redundant with https://github.com/sframe-wg/sframe/pull/68.
Given https://github.com/sframe-wg/sframe/pull/68 is already approved, can we merge it and rebase this one on top of it?

File: draft-omara-sframe.md (hunk 1 above)
Comment (bifurcation) [reply draft-omara-sframe.md:461]:
```suggestion
| Value  | Name                           | Nh | Nk | Nn |Auth Tag Len (bits) | Reference |
```

File: draft-omara-sframe.md (hunk 2 above)
Comment (bifurcation) [reply draft-omara-sframe.md:466]:
```suggestion
| 0x0004 | AES\_GCM\_256\_SHA512          | 64 | 32 | 12 |         128 | RFC XXXX  |
```

**Review by bifurcation — APPROVED** [review]

**Comment by bifurcation** [comment]
Closing in favor of #91

**Pull request closed without merging**